lib/resources/dstu2/soarian_procedure.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: Procedure
+field_name_base_url: http://hl7.org/fhir/DSTU2/procedure-definitions.html#Procedure
+fields:
+- name: code
+  type: CodeableConcept
+  binding:
+    description: A code to identify a specific procedure
+    terminology:
+    - display: Procedure Codes (SNOMED CT)
```

Comment:
All three are supported in Soarian but for any given system only one can be returned as specified by the Client, the example shows only SNOMED. We will add the same in Implementation Notes.